### PR TITLE
Feature navigation

### PR DESCRIPTION
**Pull request by modsmoss**
This branch finishes the following from #71:
- Static markup version - no functionality is added yet
- Simple dropdown for xs screens. Bootstraps native toggle collapse menu. 
  - _bootstrap js collapse package_ was added to meteor
- Added meta tag for media queries to work:

``` html
<meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

TODO: What is left of #71
Add functionality such as page links, search, share and language. - @refugeetech/platformteam 
Add fonts. @davidpaulsson is waiting for answer from the type agency.
Refine design of the toggle collapse menu  - @refugeetech/design

*_This branch is branched from feature/shrinking-navbar #94 *_

### CODE REVIEW

File: main/client/header/navbar.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Meteor.startup(function() {
+
+  // Adds class .shrinked to #top-nav
+  $(window).scroll(function() {
+    let topNav = $('nav#top-nav');
+    if ($(document).scrollTop() > 75) topNav.addClass('shrinked');
```
Comment (brylie) [inline main/client/header/navbar.js]: Add comments to these lines, describing their purpose in plain English. This supports us when we review and revise the code.

File: main/client/mainLayout.html [hunk 2]
```diff
@@ -1,3 +1,8 @@
+<head>
+    <title>Refugee Tech</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="//f.fontdeck.com/s/css/438QB1C59GkIxOiyjaJyqETvpBo/refugeetech.com/20575.css" type="text/css" />
```
Comment (brylie) [inline main/client/mainLayout.html]: What is the purpose of this external CSS? Lets avoid loading scripts from third parties where possible. Instead, place the CSS in our project.

File: main/client/header/navbar.js (hunk 1 above)
Comment (modsmoss) [inline main/client/header/navbar.js]: Sure, I'll try to make better comments. What's the procedure to update a pull request after code review? Should I just keep commiting to the same branch?

File: main/client/mainLayout.html (hunk 2 above)
Comment (modsmoss) [inline main/client/mainLayout.html]: The purpose was to fetch the fonts. This didn't work so it should have been removed before the pull request. Sorry about that

File: main/client/header/navbar.js (hunk 1 above)
Comment (leosa83) [inline main/client/header/navbar.js]: Yes @modsmoss just commit to the same branch, thanks

**Comment by leosa83** [comment]
@modsmoss
I updated the description

**Comment by modsmoss** [comment]
Thanks @leosa83

**Comment by leosa83** [comment]
Great work, i will resolve the conflicts and merge this asap.
